guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -321,14 +321,23 @@ Let's start with the HTML we're working with:
 <button type="button">Pause</button>
 ```
 
-Next, let's add an event handler to the `Play` button:
+It might be tempting to try to handle this by managing the whole set of interactions at the component level. But there are three reasons we might want to tackle this using a modifier instead:
```

Comment:
Initially, I found the 3 reasons to be confusing (a bit too abstract). It wasn't until I reached the end of this subsection, where you wrote the tagline,

```
The component manages the state. The modifier manages the DOM.
```

that I understood what you wanted readers to do.

I think it'd be good to include these lines in the beginning, then reiterate them at the end.

```suggestion
It might be tempting to define all interactions in the component class. However, as we will see soon, the key to a better solution is to separate concerns. Let the component manage the state and the modifier manage the DOM.

In general, there are 3 reasons why we might want to introduce a modifier:
```

Reply:
Good callout! Committed that suggestion directly.

Reply:
As I'm mulling, I'm going to push a slight tweak here, but the overall direction I'm still very much 👍 on.